build.fsx

```diff
@@ -75,7 +63,7 @@ let (|AnyNetcoreRuntime|_|) r =
   | FSACRuntime.NETCoreFDD -> Some ()
   | FSACRuntime.NET -> None
 
-let isTestSkipped cfg fn =
+let isTestSkipped cfg (fn: string) =
```

Comment:
Spanatee the manatee says hi

Reply:
tell me about it :D  I hit this on every project these days. and each new mono upgrade is a game of 'discover the new overloads!'